daemons/based/based_common.c

```diff
@@ -90,17 +90,13 @@ static int
 cib_prepare_diff(xmlNode * request, xmlNode ** data, const char **section)
 {
     xmlNode *input_fragment = NULL;
-    const char *update = crm_element_value(request, F_CIB_GLOBAL_UPDATE);
 
     *data = NULL;
     *section = NULL;
 
-    if (crm_is_true(update)) {
-        input_fragment = get_message_xml(request, F_CIB_UPDATE_DIFF);
-
-    } else {
-        input_fragment = get_message_xml(request, F_CIB_CALLDATA);
-    }
+    input_fragment = pcmk__xe_get_bool_attr(request,F_CIB_GLOBAL_UPDATE) ?
+                         get_message_xml(request, F_CIB_UPDATE_DIFF) :
+                         get_message_xml(request, F_CIB_CALLDATA);
```

Comment:
Breaking ternary expressions on multiple lines has confused people in the past, so better to leave as if/else

Reply:
Fixed.